src/System.Management.Automation/engine/remoting/common/remotingexceptions.cs

```diff
@@ -67,7 +67,7 @@ internal enum PSRemotingErrorId : uint
         MissingIsStartFragment = 409,
         MissingProperty = 410,
         ObjectIdsNotMatching = 411,
-        FragmetIdsNotInSequence = 412,
```

Comment:
Oops, sorry. I tried to split such things out, I guess I missed this correlation.
I have a non public commit:
`spelling: public identifiers in src/System.Management.Automation`
which is where I'll send this.